ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -403,16 +403,11 @@ public View getBannerView() {
     private boolean hasBannerSizeAd(Context context, AdSize adSize) {
         AdConfig.AdSize adSizeType = null;
 
-        int width = adSize.getWidth();
-        if (width < 0) {
-            float density = context.getResources().getDisplayMetrics().density;
-            width = Math.round(adSize.getWidthInPixels(context) / density);
-        }
-
-        ArrayList<AdSize> potentials = new ArrayList<>(3);
-        potentials.add(0, new AdSize(width, 50));
-        potentials.add(1, new AdSize(width, 90));
-        potentials.add(2, new AdSize(width, 250));
+        ArrayList<AdSize> potentials = new ArrayList<>(4);
+        potentials.add(0, new AdSize(300, 50));

```

Comment:
apart from our banner short, can we use the default enum for AdMob's AdSize for other sizes

Reply:
fixed.